metricbeat/docker-compose.yml

```diff
@@ -103,11 +103,11 @@ services:
       - 9200
 
   envoyproxy:
-    image: docker.elastic.co/observability-ci/beats-integration-envoyproxy:${ENVOYPROXY_VERSION:-v1.7.0}-1
+    image: docker.elastic.co/observability-ci/beats-integration-envoyproxy:${ENVOYPROXY_VERSION:-v1.12.0}-1
```

Comment:
I think we should keep testing against both the old and the newest version, like what we do in other modules like [redis](https://github.com/elastic/beats/blob/fc3b51a864533d8100814dab33f75825be16284a/metricbeat/tests/system/test_redis.py#L164)